dataland-frontend/src/components/pages/indices/IndexCard.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <Card class="bg-white d-card mr-2 cursor-pointer" >
+    <template #stockIndex></template>
+    <template #content>
+      <div class="grid align-items-center">
+        <div class="col-12 text-left pb-0">
+        <strong>{{ stockIndexObject[stockIndex] }}</strong>
+        </div>
+        <div class="col-5 md:col-7 text-left pt-0 text-gray-800">
+          <p>Green asset ratio</p>
+        </div>
+        <div class="col-6 md:col-5 text-right text-green-500 pt-0">
+          <span class="font-semibold text-xl">{{ green_asset_ratio }}</span>
+          <span>%</span>
+        </div>
+      </div>
+        <ProgressBar :value="green_asset_ratio" :showValue="false" class="bg-black-alpha-20 d-progressbar">
+        </ProgressBar>
+    </template>
+  </Card>
+</template>
+
+<script>
+import Card from "primevue/card";
+import ProgressBar from 'primevue/progressbar';
+import {stockIndexObject} from "@/utils/indexMapper";
+
+export default {
+  name: "IndexCard",
+  components: {Card, ProgressBar},
+  data(){
+    return {
+      stockIndexObject: stockIndexObject()
+    }
+  },
+  props: {
+    stockIndex: {
+      default: "Dax",
+      type: String
+    },
+    green_asset_ratio: {
+      default: 50,
```

Comment:
Im Backend is ein green asset ratio immer als zwischen 0 und 1 definiert. Falls hier auch, dann sollte der default vllt auf einen anderen Wert gesetzt werden?

Reply:
(Außer natürlich man coder hier quasi direkt den Prozent-Wert rein, dann ist es natürlich ok.)

Reply:
defaultwerte finde ich hier nicht so sinnvoll wenn es man überhaubt braucht